web/classroom_views.py

```
@@ -0,0 +1,101 @@
+import json
+from django.shortcuts import render, get_object_or_404
```

Ruff (0.8.2)

`django.shortcuts.get_object_or_404` imported but unused

Remove unused import: `django.shortcuts.get_object_or_404`

(F401)